### DISCUSSION TITLE
Run Lynis against a rootfs

### DISCUSSION THREAD

**Discussion by frakman1** [CISOfy/lynis#1468] [Q&A] [answered]
The product in question is a cross-compiled custom, linux-based OS that runs on a low powered ARM platform (e.g. modem, router, IoT device). Such a platform won't be able to run `lynis` natively. Therefore, it would be ideal if `lynis` can run on a host with the option of pointing it to a rootfs directory containing the target OS. 

Is this a feature that can be supported or do you have any suggestions on how to accomplish this?

**Comment by mboelen** [comment, marked as answer]
It will be possible in the future, as preparations are made to scan a specific path. However, all tests need to be adopted to use this path. The related 'hidden' option is `--rootdir` and not advertised yet.

If you like to help make things possible, I suggest diving into the code and see what we can do together to get this feature fully implemented (including a good way to test things).
